src/Thenewboston/Bank/Api/InvalidBlocksService.cs

```diff
@@ -18,7 +18,7 @@ public InvalidBlocksService(IHttpRequestSender requestSender)
             _requestSender = requestSender;
         }
         
-        public async Task<IEnumerable<BankInvalidBlock>> GetInvalidBankBlocks()
+        public async Task<IEnumerable<BankInvalidBlock>> GetInvalidBankBlocksAsync()
         {
             try
```

Comment:
Please remove Try/Catch blocks. Throughout our other service models we have used guard clauses which throw specific exceptions that the developer should handle. We will be creating the specific exceptions at a later stage, and would encourage you to join in on the discussion of how those should be laid out. I anticipate that this is being done to reduce the complexity (indentation level) of the code and make it easier to maintain.

Reply:
Removed try catch blocks